codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -36,7 +36,7 @@ public int generateHashCode(CharSequence name) {
         }
     };
 
-    private static final HashCodeGenerator<CharSequence> CHARSEQUECE_CASE_SENSITIVE_HASH_CODE_GENERATOR =
+    private static final HashCodeGenerator<CharSequence> CHARSEQUENCE_CASE_SENSITIVE_HASH_CODE_GENERATOR =
```

Comment:
@trustin this stuff will get removed (at least that's my proposal :)) with a new headers implementation I ll hopefully be able to open a PR for tomorrow. So doesn't matter really :-).